package_manifest.yaml

```diff
@@ -592,6 +592,8 @@ katello_packages:
         - "el8-foreman-{{ foreman_version }}"
         - "el8-foreman-plugins-{{ foreman_version }}"
         - "el8-katello-candlepin-{{ katello_version }}"
+  children:
+    satellite_packages: {}
```

Comment:
This is the slightly yucky part being stuck under here, but I assume you are doing so to prevent needing another repository since it needs katello.

Reply:
Well, it doesn't *need* katello, only none-core dep is deface, so I could also put it into plugins.
This was mostly so I don't have to create an own tag to build into.